Jenkinsfile

```diff
@@ -3,7 +3,6 @@
 def channel = "${env.BRANCH_NAME}".contains('master') ? '#ksqldb-quality-oncall' : '#ksqldb-warn'
 
 common {
-    nodeLabel = 'docker-openjdk11'
```

Comment:
The issue with using the default node label is that PRs are build with a different Java version than the non-PR builds. So we might get in a situation where a PR build is green but a non-PR build fails.
https://github.com/confluentinc/jenkins-common/blob/be2b26b82b10dd9a901c0d8538c96274ac5dd16a/vars/confluentBranchJdkVersion.groovy#L2

For now, we should merge these PR as it is and then in a second step align PR and non-PR builds by specifying node labels.